app/upgrades/v16/upgrades.go

```diff
@@ -21,7 +21,38 @@ func CreateUpgradeHandler(
 			return vm, err
 		}
 
+		err = ConfigureFeeMarketModule(ctx, keepers)
+		if err != nil {
+			return vm, err
+		}
+
 		ctx.Logger().Info("Upgrade complete")
 		return vm, err
 	}
 }
+
+func ConfigureFeeMarketModule(ctx sdk.Context, keepers *keepers.AppKeepers) error {
+	params, err := keepers.FeeMarketKeeper.GetParams(ctx)
+	if err != nil {
+		return err
+	}
+
+	params.Enabled = true
+	params.FeeDenom = "uatom"
+	// TODO check and set param values:
```

Comment:
before this PR is marked as ready, we need to decide what are wanted values for feemarket upgrade params